pkg/service/backup/worker_clustercheck.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2022 ScyllaDB
+
+package backup
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/scylladb/scylla-manager/v3/pkg/util/retry"
+)
+
+func (w *worker) CheckCluster(ctx context.Context, hosts []hostInfo, clusterCheckBackoff retry.Backoff) error {
+	notify := func(err error, wait time.Duration) {
+		w.Logger.Info(ctx, "Current load of one of the nodes is above the configured threshold...", "error", err, "wait", wait)
+	}
+
+	return retry.WithNotify(ctx, func() error {
+		for _, h := range hosts {
+			load, err := w.Client.CurrentLoad(ctx, h.IP)
+			if err != nil {
+				return retry.Permanent(errors.Wrap(err, "node's load metric"))
+			}
+			if load >= w.Config.MaxLoadThreshold {
+				return errors.Errorf("load %s on %s node is above configured threshold equal to %s",
+					strconv.FormatFloat(load, 'f', 2, 64),
+					h.IP,
+					strconv.FormatFloat(w.Config.MaxLoadThreshold, 'f', 2, 64))
```

Comment:
Could I have one nitpick? Why not utilize float conversion as a part of `Errotf`?

```suggestion
				return errors.Errorf("load %.2f on %s node is above configured threshold equal to %.2f",	load, h.IP, w.Config.MaxLoadThreshold)
```

Reply:
I believe `%.2f` should do the trick

Reply:
@mrVanboy of course you could and thanks for the comment. Will do.
@mmatczuk you're alive ! :) How are the things going ? Wanna go back :) ?

Reply:
Done